feature/experimental/p4rt/tests/p4rt_election/p4rt_election_test.go

```diff
@@ -316,21 +316,27 @@ func TestZeroMaster(t *testing.T) {
 		highID:     inId0,
 		handle:     clientConnection(t, dut),
 		deviceID:   deviceId,
-		wantFail:   true,
-		wantStatus: 3,
+		wantFail:   false,
+		wantWrite:  false,
+		wantRead:   true,
+		wantStatus: 0,
 	}
 	t.Run(test.desc, func(t *testing.T) {
 		resp, err := streamP4RTArb(&test)
-		if err == nil && test.wantFail {
-			t.Errorf("Zero ElectionID (0,0) is allowed: %v", err)
+		if err != nil && test.wantFail {
+			t.Errorf("Zero ElectionID (0,0) is rejected by P4RT server: %v", err)
 			removeClient(test.handle)
 		} else {
-			t.Logf("Zero ElectionID (0,0) connection failed as expected: %v", err)
+			t.Logf("Zero ElectionID (0,0) connection success as expected: %v", err)
```

Comment:
Remove the `else` statement here and add a `return` inside the if statement.

Reply:
@Ankur19 ,
Does this look ok?

`t.Run(test.desc, func(t *testing.T) {
		resp, err := streamP4RTArb(&test)
		if err != nil && test.wantFail {
			t.Errorf("Zero ElectionID (0,0) is rejected by P4RT server: %v", err)
		} 
		// Validate status code
		if resp != test.wantStatus {
			t.Errorf("Incorrect status code received: want %d, got %d", test.wantStatus, resp)
		}
		// Validate the response for Read/Write for zero election ID
		validateRWResp(t, &test)
		// Disconnect Primary
		removeClient(test.handle)
	}`

Reply:
With t.Errorf do we need a separate return statement?

Reply:
```
t.Run(test.desc, func(t *testing.T) {
		resp, err := streamP4RTArb(&test)
		if err != nil && test.wantFail {
			t.Errorf("Zero ElectionID (0,0) is rejected by P4RT server: %v", err)
			removeClient(test.handle)
                        return
		} 
		t.Logf("Zero ElectionID (0,0) connection success as expected: %v", err)
                ....
```

Reply:
Thanks i have taken care of this.